orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
I hope they wouldn't! If they did though, then it would fail if we changed the type of CacheType.

If they use the constant as you suggest, then we can change the type of the constant regardless of whether it is aliased to `CacheType` or a bare int or string or anything else.